Week5/app/src/main/res/layout-land/fragment_profile.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
```

Comment:
I think it would be better if you built this layout using CosntraintLayout. This is just too much unnecessary nesting